includes/actions/class-amp-frontend-actions.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Callbacks for adding AMP-related things to the main theme
+
+require_once( AMP__DIR__ . '/includes/actions/class-amp-actions.php' );
+
+class AMP_Frontend_Actions {
+	
+	public static function register_hooks() {
+		add_action( 'wp_head', 'AMP_Frontend_Actions::add_canonical' );
+	}
+	
+	public static function add_canonical() {
+		if ( false === apply_filters( 'add_canonical_link', true ) ) {
```

Comment:
This change here renamed the filter from `amp_frontend_show_canonical` to `add_canonical_link`. This needs to be reverted. I think this needs to be reverted.